third_party/gloo/gloo.BUILD

```diff
@@ -22,7 +22,7 @@ substitions = {
     "#cmakedefine01 GLOO_USE_REDIS": "#define GLOO_USE_REDIS 0",
     "#cmakedefine01 GLOO_USE_IBVERBS": "#define GLOO_USE_IBVERBS 0",
     "#cmakedefine01 GLOO_USE_MPI": "#define GLOO_USE_MPI 0",
-    "#cmakedefine01 GLOO_USE_LIBUV": "#define GLOO_USE_LIBUV 0",
+    "#cmakedefine01 GLOO_USE_LIBUV": "#define GLOO_USE_LIBUV (__APPLE__ ? 1 : )",
```

Comment:
Should we keep the original `0`?
```suggestion
    "#cmakedefine01 GLOO_USE_LIBUV": "#define GLOO_USE_LIBUV (__APPLE__ ? 1 : 0)",
```
Alternatively, would `"#define GLOO_USE_LIBUV __APPLE__"` have the same effect?

Reply:
Thanks! Leaving out the `0` was an oversight.

Re: `#define GLOO_USE_LIBUV __APPLE__` I'm not sure. I think `__APPLE__` is `1` when it is defined, but it's probably not `0` otherwise.

The other question is whether any of this is strictly necessary, running `git grep USE_LIBUV` in https://github.com/facebookincubator/gloo shows hits only in cmake files and `gloo/config.h.in`, but none in actual code.

Reply:
Got it. Thank you for investigating!